src/components/Player/Player.component.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useContext, useEffect, useState } from 'react';
+import styled, { keyframes } from 'styled-components';
+import { useAuth } from '../../providers/Auth/Auth.provider';
+import { VideoContext } from '../../providers/Video/Video.provider';
+import { FavoritesContext } from '../../providers/Favorites/Favorites.provider';
+
+const PlayerContainer = styled.div`
+  height: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-top: 3px;
+  max-width: 1200px;
+  min-width: 1200px;
```

Comment:
You could add a % to the width to make it responsive

Reply:
thanks will do